Userland/Libraries/LibC/scanf.cpp

```diff
@@ -313,8 +313,10 @@ struct ReadElement<char*, ReadKind::Normal> {
         if (str.is_empty())
             return false;
 
-        memcpy(ptr, str.characters_without_null_termination(), str.length());
-        ptr[str.length()] = 0;
+        if (ptr) {
```

Comment:
What if user will provide a null pointer as an argument to the `scanf`, but will not include an assignment suppression modifier? Previously it would simply crash the program, but now, if I understand the code correctly, it will just silently skip this argument. Not sure if this is a desired behavior.

Reply:
@TSultanov You are correct here. Scanf should crash under this situation. I will rework the patch.